Source/WebCore/Modules/webauthn/WebAuthenticationUtils.cpp

```diff
@@ -158,7 +158,7 @@ Vector<uint8_t> buildAttestationObject(Vector<uint8_t>&& authData, String&& form
 }
 
 // FIXME(181948): Add token binding ID.
-Ref<ArrayBuffer> buildClientDataJson(ClientDataType type, const BufferSource& challenge, const SecurityOrigin& origin, WebAuthn::Scope scope)
+Ref<ArrayBuffer> buildClientDataJson(ClientDataType type, const BufferSource& challenge, const SecurityOrigin& origin, WebAuthn::Scope scope, std::optional<WTF::String> topOrigin)
```

Comment:
We almost never use `std::optional<String>` in WebKit because String already has a null state. As a result, you can pass a `const String& topOrigin` here and check `topOrigin.isNull()`.